org.eclipse.ice.modeling/src/main/java/org/eclipse/ice/modeling/states/State.java

```diff
@@ -0,0 +1,10 @@
+package org.eclipse.ice.modeling.states;
+
+/**
```

Comment:
We should coordinate this with the [Commands#CommandStatus Enum](https://github.com/eclipse/ice/blob/jay%2FMarkIII/org.eclipse.ice.commands/src/main/java/org/eclipse/ice/commands/CommandStatus.java), so that we have a consistent set of states that every step of the workflow can take. We can modify Commands and this so that they are consistent with one another.

Reply:
Yes, and I need to learn how to do enums in Java.  This was just an initial place holder so I did not loose my thinking.

Reply:
@osbornjd We should ask ourselves whether or not it is necessary to reuse the CommandStatus enum from Commands for two reasons:
1) That's not a lot of code to justify a whole set of dependencies.
2) It may be that they enumerations are close but not exact.